src/Components/LetterEncoding/LetterEncoding.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react'
+import Anime, { anime } from 'react-anime'
+
+    function LetterButton(props) {
+        return (
+            <button className="button button-height is-static has-text-weight-bold is-family-secondary px-4 is-size-2"
+                style={props.shouldHandleMouseEnter ? {pointerEvents: "auto"} : {}}
+                onMouseEnter={props.shouldHandleMouseEnter ? props.handleMouseEnter : null}>
+                {props.children}
+            </button>
+        );
+    }
+
+class LetterEncoding extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            showLetter: false
+        };
+    }
+
+    handleMouseEnter = () => {
+        if (!this.state.showLetter) {
+            this.setState({
+                showLetter: true
+            });
+            this.animateArrow();
+        }
+    }
+
+    animateArrow = () => {
+        anime({
+            targets: [`#arrow${this.props.num}`],
+            x2: '98%',
+            duration: 800,
+            easing: "easeOutCubic"
+        });
+    }

```

Comment:
This isn't anything that will need changing on account of the fact that we won't have to worry about ID collision in our webapp, but there is something that you can do to make your code more maintainable for others down the line or in larger projects. Instead of using an ID selector and the `anime()` function is wrap the component you want to animate in an `<Anime />` component that only renders when the particular prop or condition is satisfied. This way, we don't need to select elements by ID, and can just let React do the work for us.

For example:

```js
import React, {useState} from 'react';
import Anime from 'react-anime';

export default function Demo() {
    const [anim, setAnim] = useState(false);
    
    return (
        <div onMouseEnter={()=>setAnim(true)}>
            <h1>Some text</h1>

            {anim &&
            <Anime translateX={'50px'} duration={800}>
                <p>And this is stuff that gets animated when we enter</p>
            </Anime>
            }
        </div>
    );
}
```

This component can be used as many times as we want on the page, and each individual animated piece of text will have its own separate trigger.